stack/__main__.py

```diff
@@ -41,6 +51,8 @@
     ['name', 'member_key', 'permissions'],
 )
 
+NON_NAME_REGEX = re.compile(r'[^A-z0-9_-]')
```

Comment:
```suggestion
NON_NAME_REGEX = re.compile(r'[^A-Za-z0-9_-]')
```

Reply:
I'm not following this one, doesn't `A-z` cover `A-Za-z`?

Reply:
Definitely, if you consider a particular ordering. However, that's not how regex character classes are typically defined: https://github.com/google/re2/wiki/Syntax

Reply:
Ooh, learn something new every day!